addons/analytic/migrations/10.0.1.1/tests/test_analytic.py

```diff
@@ -0,0 +1,13 @@
+# coding: utf-8
+from openerp.tests.common import TransactionCase
+
+
+class TestAnalytic(TransactionCase)
```

Comment:
I have just discovered that this is not being executed, or this would be marked as error, as it misses the final `:`

cc @hbrunn @gfcapalbo